sandbox/src/EventMerge.js

```diff
@@ -4,12 +4,12 @@ export default function EventMerge() {
   const eventMergeIdPromise = useRef(window.alloy("createEventMergeId"));
 
   useEffect(() => {
-    eventMergeIdPromise.current.then(eventMergeId => {
+    eventMergeIdPromise.current.then(result => {
```

Comment:
This is a separate fix for something else?

Reply:
Yeah, I thought I left a comment somewhere but I don't know where it is now. The event merge ID view in the sandbox was failing because the promise is resolved with an object containing the event merge ID rather than the event merge ID itself.